keyboards/keebio/iris/keymaps/csc027/rules.mk

```diff
@@ -0,0 +1,15 @@
+API_SYSEX_ENABLE = no
+AUDIO_ENABLE = no
+BACKLIGHT_ENABLE = yes
+BLUETOOTH_ENABLE = no
+BOOTMAGIC_ENABLE = no
+COMMAND_ENABLE = no
+CONSOLE_ENABLE = no
+EXTRAKEY_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+MIDI_ENABLE = no
+MOUSEKEY_ENABLE = yes
+NKRO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+SLEEP_LED_ENABLE = no
+UNICODE_ENABLE = no
```

Comment:
This can be condensed down to:
```make
BACKLIGHT_ENABLE = yes
CONSOLE_ENABLE = no
EXTRAKEY_ENABLE = no
LTO_ENABLE = yes
MOUSEKEY_ENABLE = yes
NKRO_ENABLE = yes
```

Reply:
Per both your suggestion, the Planck and Iris rules files have been condensed.